wowchemy/assets/scss/wowchemy/layouts/_book.scss

```diff
@@ -258,18 +265,21 @@ ul.toc-top {
   font-size: 16px; // Lighthouse: min tap target size
 }
 
+// stylelint-disable-next-line selector-id-pattern
 .dark #TableOfContents li a,
 .dark .toc-top li a {
   color: rgba(255, 255, 255, 0.65);
 }
 
+// stylelint-disable-next-line selector-id-pattern
 #TableOfContents li a:hover,
 .toc-top li a:hover {
   color: $sta-primary;
   text-decoration: none;
 }
 
 /* ScrollSpy active link style. */
+// stylelint-disable-next-line selector-id-pattern
 #TableOfContents li a.active {
   color: $sta-primary;
   font-weight: 700;
```

Comment:
These come from Hugo, so we don't have control over the style of the `id`, thus disable. Could perhaps be done with a single ``stylelint-disable .. stylelint-enable` scope, if you'd prefer